### ISSUE TITLE
I don’t know when to converge, because the loss value cannot be visualized during training

### ISSUE THREAD

**Issue by LxyoryxL** [Trusted-AI/adversarial-robustness-toolbox#1084]
question:
- [ ] In last line, I don’t know how to set the epoch, because the loss value cannot be visualized during training, and I don’t know when to converge.

here is my code......
```
 ###  Create the model
    cnn_model1=CNN_model1()

 ###  Define the loss function and the optimizer
    criterion = nn.CrossEntropyLoss()
    optimizer = optim.Adam(cnn_model1.parameters(), lr=0.001)

 ###  Create the ART classifier
    classifier = PyTorchClassifier(
        model=cnn_model1,
        loss=criterion,
        optimizer=optimizer,
        input_shape=(3,image_width,  image_height),
        nb_classes=2,
        device_type='gpu'
    )

  ### Train the ART classifier
    classifier.fit(x_train, y_train, batch_size=60, nb_epochs=20)
```

**Comment by beat-buesser** [comment]
Hi @LxyoryxL That's true, the method `PyTorchClassifier.fit` is a very basic fit method for very basic applications. For cases with more advanced training, we recommend to train the model, `cnn_model1` in your case outside of ART using the framework's methods for model training, this should give you the full flexibility of the framework for training models. After training, you can then provide the trained model to `PyTorchClassifier`.
